sui-execution/v0/sui-move-natives/src/dynamic_field.rs

```diff
@@ -79,7 +79,7 @@ pub struct DynamicFieldHashTypeAndKeyCostParams {
  *              + dynamic_field_hash_type_and_key_value_cost_per_byte * size_of(k)  | covers cost of operating on the value `k`
  *              + dynamic_field_hash_type_and_key_type_tag_cost_per_byte * size_of(type_tag(k))    | covers cost of operating on the type tag of `K`
  **************************************************************************************************/
-#[instrument(level = "trace", skip_all, err)]
+#[instrument(level = "trace", skip_all)]
```

Comment:
Do we need to touch the old versions?

Reply:
IMO we should. We don't _have_ to, but then folks will hit this when e.g., synching from genesis or from an older epoch. So we don't need to do this, but it's the right thing to do for consistency and for noise reduction when synching older epochs.